Vagrantfile

```diff
@@ -6,10 +6,15 @@
 require 'yaml'
 
 current_dir    = File.dirname(File.expand_path(__FILE__))
-configfile        = YAML.load_file("#{current_dir}/.vagrantconfig.yml")
-yaml_config = configfile['configs']['dev']
+configfile     = YAML.load_file("#{current_dir}/.vagrantconfig.yml")
+travisfile     = YAML.load_file("#{current_dir}/.travis.yml")
+yaml_config    = configfile['configs']['dev']
 
 Vagrant.configure(2) do |config|
+  # This trick is used to prefer a VM box over docker
```

Comment:
This didn't work. I did a pull, and get the same error as before (on both vagrant destroy and up):
```
D:\Github\testdelete\qgroundcontrol>vagrant up
A Docker command executed by Vagrant didn't complete successfully!
The command run along with the output from the command is shown
below.

Command: ["docker", "ps", "-a", "-q", "--no-trunc", {:notify=>[:stdout, :stderr]}]

Stderr: error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.28/containers/json?all=1: open //./pipe/docker_engine: The system cannot find the file specified. In the default dae
 be run elevated to connect. This error may also indicate that the docker daemon is not running.


Stdout:
```